src/ellipse.rs

```diff
@@ -117,7 +117,7 @@ impl Ellipse {
 
     /// Returns the two radii of this ellipse.
     ///
-    /// The first number is the horizontal radius and the second is the vertical
+    /// The first number is the horizontal radius, and the second is the vertical
```

Comment:
```suggestion
    /// The first number is the horizontal radius and the second is the vertical
```

The trailing comma is to join the dependent clause "before rotation", but a simple "and" works here